.env

```diff
@@ -1,4 +1,6 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```

Comment:
This file needs to be reset to latest 

@pjlee11 Looks like your git trick isn't working, can you maybe raise an issue to take a look at it please?